notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<p>I think score difference aggregated across all categories by severity type and for multiple metrics would be interesting. Possibly as a table again e.g.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">| Metric      | Minor | Major | Critical |
| ----------- | ----- | ----- | -------- |
| BLASER      |       |       |          |
| COMET       |       |       |          |
| ChrF        |       |       |          |
</pre><p> </p><p>and maybe separately vs. language if it adds anything interesting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>